zcash/src/public_key.rs

```diff
@@ -11,8 +11,18 @@ use std::str::FromStr;
 use std::{fmt, fmt::Display};
 use zcash_primitives::{keys::FullViewingKey, JUBJUB};
 
+use byteorder::{BigEndian, ByteOrder};
+use crypto::sha2::sha256_digest_block;
+
+use crate::SproutSpendingKey;
+use base58::{FromBase58, ToBase58};
 use sapling_crypto::jubjub::JubjubBls12;
 
+//#[allow(unreadable_literal)]
+static H256: [u32; 8] = [
```

Comment:
Why is this necessary? `crypto::sha2` provides it: https://github.com/DaGenix/rust-crypto/blob/cc1a5fde1ce957bd1a8a2e30169443cdb4780111/src/sha2.rs#L1133